keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -967,8 +959,8 @@ devices:
         int: 3
       uuid:
         string: GPU-662077db-fa3f-0d8f-9502-21ab0ef058a2
-    consumesCapacity:
-    - capacityPool: gpu-3-pool
+    consumesCounter:
```

Comment:
`consumesCounters` (plural)

throughout the file

Reply:
I'll leave it as consumesCounter to match what we have in implementation. The v1beta2 rename should be out of the scope of this KEP.